### DISCUSSION TITLE
regolith 3 swapping Mod1 and Mod4

### DISCUSSION THREAD

**Discussion by jsundahl** [regolith-linux/regolith-desktop#1064] [Q&A]
My versions:
```
--- /etc/regolith/version:
REGOLITH_VERSION=3.2
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=24.04
```

I'm trying to swap Mod1 and Mod4 as described here https://regolith-desktop.com/docs/using-regolith/configuration/#keybindings. However, when I do so and log out/in I get many errors like:
```
ERROR: Duplicate keybinding in config file:
  state mask 0x8 with keysym 1, command "move container to workspace number 1; workspace number 1"
```

I uninstalled regolith 2 and wiped .config/regolith before reinstallling regolith 3. I have not changed any values in any of the configs.

I'd be very grateful for any advice.

**Comment by jsundahl** [comment]
I should add that everything seems to be working fine despite the errors.

**Reply by abacao** [reply]
Check out what files are being loaded in your configuration by
```
i3 --more-version
```

Was `~/.config/regolith3/Xresources` the only file you have changed?
